Formula/libtommath.rb

```diff
@@ -15,6 +15,8 @@ class Libtommath < Formula
     sha256 cellar: :any_skip_relocation, high_sierra:   "26e39af069485ef58c3517fb765db3a5e8dba0f253aac3d0d5968ff2a35e595b"
   end
 
+  depends_on "libtool" => :build
```

Comment:
Actually, this just occurred to me: Homebrew `libtool` binaries are named `glibtool` and `glibtoolize`. Does `make` not need help finding them? 

I'm actually wondering if it's possible that `make` didn't find them, but was able to build everything just fine with system `libtool`.

Reply:
`libtommath` makefile includes [special case for darwin platform](https://github.com/libtom/libtommath/blob/develop/makefile.shared).

```
ifeq ($(PLATFORM), Darwin)
  LIBTOOL:=glibtool
else
  LIBTOOL:=libtool
endif
```

We can see that `libtool` is required.